components/date-picker/RangePicker.razor.cs

```diff
@@ -212,34 +212,6 @@ private async Task OnInputClick(int index)
             }
         }
 
-        protected void OnInput(ChangeEventArgs args, int index = 0)
-        {
-            if (args == null)
-            {
-                return;
-            }
-
-            if (!_duringManualInput)
-            {
-                _duringManualInput = true;
-            }
-
-            if (FormatAnalyzer.TryPickerStringConvert(args.Value.ToString(), out DateTime parsedValue, false))
-            {
-                if (IsDisabledDate(parsedValue))
-                {
-                    return;
-                }
-
-                _pickerStatus[index].SelectedValue = parsedValue;
-                ChangePickerValue(parsedValue, index);
-            }
-            else
-            {
-                _pickerStatus[index].SelectedValue = null;
-            }
```

Comment:
I noticed this. Did we miss it?

Reply:
This is add from your previous PR https://github.com/ant-design-blazor/ant-design-blazor/pull/3087/files#diff-9cc53059c159df4e894628ddb114efb40c2bfec6a051992a57a8a31b16215dd1R239

Reply:
Yes, thanks, I'm trying to fix it. The tests are not repeatable.  I'll stabilize it

Reply:
I did it, but with some other changes. At now start tests multiple times with multithreading does not fail on my machine.

Reply:
@ElderJames test coverage has dropped, what do you think about  async methods in DatePicker, I have to revert this or use async overloads in tests?

Reply:
I reverted changes with async

Reply:
Thank you